deploy/example/backup-app/README.md

```diff
@@ -0,0 +1,56 @@
+# Example Backup Application
+
+This document provides steps to install a backup application that communicates with the snapshot-metadata service to utilize the Changed Block Tracking (CBT) feature. You can use this example as a reference when deploying the backup application with the required permissions to use the CBT feature.
+
+## Prerequisites:
+
+- Installed ClusterRoles and CRDs: Follow the instructions in this [guide](../../README.md) to ensure these components are installed.
+- Installed Snapshot-Metadata Service with a CSI Driver: Refer to this [guide](../csi-driver/README.md) to install the snapshot-metadata service using a dummy CSI driver.
```

Comment:
Why is this a prerequisite?

Reply:
My intention was to document e2e example, but as suggested we can just keep common components